package.json

```diff
@@ -111,7 +111,7 @@
   "lint-staged": {
     "*.js": [
       "eslint --cache --fix",
-      "prettier --write"
+      "prettier --ignore-path .gitignore --write ."
```

Comment:
Updating prettier hook and ignore gitignore

Reply:
So the prettier never worked because we didn't specify the path?

Reply:
yup

Reply:
I may also be worth ignoring the build_helpers & markdown files

Reply:
Done